VTUL/dlp-access
# Migrate dev environment to the PPRD account

**Migrate dev environments to the PPRD account.**
* * *

**JIRA Ticket**: ([LIBTD-2669](https://webapps.es.vt.edu/jira/browse/LIBTD-2669)) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Migrate dev environments to the PPRD account.a

# What's the changes? (:star:)
A in-depth description of the changes made by this PR. Technical details and possible side effects.
* New app in PPRD
* New backend env
* Handle user-groups as all lowercase
* fix tests (mostly to remove references to :nth-child() which make the site much safe


# How should this be tested?
* Check preview ( https://pprd-migration.d2ysrrdhih4bgc.amplifyapp.com/ ) to ensure that all tests pass in PPRD account for this env.
* Check that everything is functioning normally.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* branch: `pprd-migration`


# Interested parties
@andreaWaldren 

(:star:) Required fields
